### DISCUSSION TITLE
Where can I view crash logs?

### DISCUSSION THREAD

**Discussion by grigoryvp** [Hammerspoon/hammerspoon#2988] [Q&A] [answered]
Recently, hammerspoon just exit without any error few times a day on all my Mac computers (both Big Sur and Monterey). Is it a way for me to see crash log so I can get a hint what's wrong?

**Comment by latenitefilms** [comment, marked as answer]
Try looking in: 

```
~/Library/Logs/DiagnosticReports
```
